meta-resin-common/recipes-connectivity/libmbim/libmbim_%.bbappend

```diff
@@ -0,0 +1,3 @@
+FILESEXTRAPATHS_append := ":${THISDIR}/files"
+
+SRC_URI_append = " file://0001-avoid-iconv.patch"
```

Comment:
Could you also have another commit for a CHANGELOG entry like this PR? https://github.com/resin-os/meta-resin/pull/653/commits